api/src/pcapi/alembic/versions/sql/schema_init.sql

```diff
@@ -5391,7 +5391,6 @@ INSERT INTO public.feature VALUES (23, 'SYNCHRONIZE_ALLOCINE', 'Permettre la syn
 INSERT INTO public.feature VALUES (24, 'USE_PRICING_POINT_FOR_PRICING', 'Utilise le modèle VenuePricingPointLink pour la valorisation', false);
 INSERT INTO public.feature VALUES (25, 'ENABLE_BOOST_API_INTEGRATION', 'Active la réservation de places de cinéma via l''API Boost', false);
 INSERT INTO public.feature VALUES (26, 'ENABLE_PRO_ACCOUNT_CREATION', 'Permettre l''inscription des comptes professionels', true);
-INSERT INTO public.feature VALUES (27, 'ENABLE_BACKOFFICE_API', 'Autorise l''accès à l''API backoffice', false);
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Ne pas changer ce fichier !
C'est la première brique des migrations

Cette ligne disparaitre le jour où on squash les migrations de nouveau

Reply:
Bon à savoir !
Je ne savais en effet pas exactement si je devais faire sauter cette ligne ou non